google/fingerprinters/web/scripts/updater/community/spark/app/docker-compose.yml

```
@@ -0,0 +1,27 @@
+services:
+  spark-master:
+    image: apache/spark:${SPARK_VERSION}
+    container_name: spark-master
+    hostname: spark-master
+    ports:
+      - "8080:8080"
+      - "4040:4040"
+    volumes:
+       - ./examples:/opt/spark/examples/src/main/python
+    environment:
+      - SPARK_MASTER_HOST=spark-master
+      - SPARK_MASTER_PORT=7077
+      - SPARK_LOCAL_HOSTNAME=127.0.0.1
```

```suggestion
      - SPARK_LOCAL_HOSTNAME=127.0.0.1
      - DEBIAN_FRONTEND=noninteractive
```
Prevents interactive prompts